backends-common/rabbitmq/src/test/java/org/apache/james/backend/rabbitmq/RabbitMQExtension.java

```diff
@@ -84,14 +91,16 @@ public RabbitMQExtension build() {
 
     private final DockerRabbitMQ rabbitMQ;
     private final Consumer<DockerRabbitMQ> afterAllStage;
+    private final boolean restartDockerEachTest;
 
     private SimpleChannelPool simpleChannelPool;
     private RabbitMQConnectionFactory connectionFactory;
     private SimpleConnectionPool connectionPool;
 
-    public RabbitMQExtension(DockerRabbitMQ rabbitMQ, Consumer<DockerRabbitMQ> afterAllStage) {
+    public RabbitMQExtension(DockerRabbitMQ rabbitMQ, Consumer<DockerRabbitMQ> afterAllStage, boolean restartDockerEachTest) {
```

Comment:
I'm a bit reluctant of using a boolean for that, but I have no simple solution for that, maybe an enum.